website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,22 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## June 2024
 
+#### dbt Semantic Layer
+- **New:** Introduced a new granularity option for cumulative metrics, `period_agg`, nested under the `type_params` parameter. It specifies how to roll up the cumulative metric to another granularity. Options are `start`, `end`, `average`. Defaults to `start` if no `window` is specified.
+- **New:** Added validation that derived metrics must always have an `expr` key.
+- **New:** Enabled cache to pickup where filters that are included in saved queries.
+- **Enhancement:** [Cumulative metrics](/docs/build/cumulative) now support the `cumulative_type_params` key. The `cumulative_type_params` parameter replaces `type_params`. MetricFlow supports the old `type_params` key but will issue a warning if used.
+- **Enhancement:** Updated MetricFlow to use the new `cumulative_type_params` fields for cumulative metrics.
+- **Enhancement:** In [Google Sheets](/docs/cloud-integrations/semantic-layer/gsheets), we added information icons and descriptions to metrics and dimensions options in the Query Builder menu. You can hover over the information icon to see a description of the metric or dimension. Available in the following Query Builder menu sections: metric, group by, where, saved selections, and saved queries.
+- **Enhancement:** You can now apply granularity to all time dimensions, not just metric time. This update uses our [APIs](/docs/dbt-cloud-apis/sl-api-overview) to support granularity selection on any chosen time dimension.
+- **Enhancement:** Improved querying error message when no semantic layer credentials were set.
+- **Fix:** Removed errors preventing querying cumulative metrics with other granularities.
+- **Fix:** Fixed various Tableau errors when querying certain metrics or when using calculated fields.
+- **Fix:** Relaxed naming field expectations to better identify calculated fields.
+- **Fix:** Fixed an error when refreshing database metadata for columns that we can't convert to Arrow. These columns will now be skipped. This mainly affected Hex users while refreshing the entire database metadata.
```

Comment:
This is just for Redshift, right? @aiguofer We might want to explain more on the outcome and table type

Reply:
Yeah I think we could mention it only affects Redshift users with custom types

Reply:
oh ok, i took the content from here: https://linear.app/dbt-labs/issue/SL-2005/skip-external-tables-in-get-schemas-to-handle-errors#comment-c8b4ffc5 which said it affected hex users. I'll update it then.